roofit/xroofit/inc/RooFit/xRooFit/xRooNode.h

```diff
@@ -421,7 +421,7 @@ class xRooNode : public TNamed, public std::vector<std::shared_ptr<xRooNode>> {
    int fTimes = 1;      // when the same comp appears multiple times in a parent node, this is increased to reflect that
    int fBinNumber = -1; // used by 'bin' nodes (a node that refers to a specific bin of a parent)
    std::shared_ptr<xRooNode> fParent; //!
-   std::string fFolder;               // folder to put this node in when 'organising' the parent
+   std::string fFolder = "";          // folder to put this node in when 'organising' the parent
```

Comment:
```suggestion
   std::string fFolder;          // folder to put this node in when 'organising' the parent
```
That's another redundant thing that `clang-tidy` will complain about.